### PR TITLE
Bug 1581061 - Include perf-alert keyword into our bug templates

### PR DESCRIPTION
**Pull request by octavian-negru**

### CODE REVIEW

File: treeherder/perf/fixtures/performance_bug_templates.yaml [hunk 1]
```diff
@@ -2,10 +2,10 @@
   pk: 1
   fields:
     framework: 1
-    keywords: perf, regression, talos-regression
-    default_component: Performance
+    keywords: perf, perf-alert, regression, talos-regression
+    default_component: General
```
Comment (davehunt) [inline treeherder/perf/fixtures/performance_bug_templates.yaml]: Is there a reason for changing the default component?

File: treeherder/perf/fixtures/performance_bug_templates.yaml [hunk 2]
```diff
@@ -2,10 +2,10 @@
...
     default_product: Testing
-    cc_list: dave.hunt@gmail.com, rwood@mozilla.com, igoldan@mozilla.com, fstrugariu@mozilla.com, alexandru.ionescu@softvision.ro, marian.raiciof@softvision.ro
+    cc_list: dave.hunt@gmail.com, rwood@mozilla.com, igoldan@mozilla.com, fstrugariu@mozilla.com
```
Comment (davehunt) [inline treeherder/perf/fixtures/performance_bug_templates.yaml]: I suspect you're on an older revision, as these were recently added.

File: treeherder/perf/fixtures/performance_bug_templates.yaml (hunk 1 above)
Comment (octavian-negru) [reply treeherder/perf/fixtures/performance_bug_templates.yaml]: I had a git problem. No, there is not a reason for changing the default component. I will solve this.

File: treeherder/perf/fixtures/performance_bug_templates.yaml (hunk 2 above)
Comment (octavian-negru) [reply treeherder/perf/fixtures/performance_bug_templates.yaml]: Oh, I forgot to rebase the currrent git branch. My bad.

**Pull request closed without merging**